### DISCUSSION TITLE
Potential breaking changes in Tor Browser and Mullvad Browser casks coming version 13.5

### DISCUSSION THREAD

**Discussion by morganava** [Homebrew/discussions#5166] [Casks]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

Ping @krehel 

Sorry, I couldn't find your email but as it seems you're the most recent maintainer of the Tor and Mullvad Browser casks I thought I'd ping you. 

With [tor-browser#42398](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/42398) and [tor-browser-build#40569](https://gitlab.torproject.org/tpo/applications/tor-browser-build/-/issues/40569) we have *potentially* altered installation/settings paths that these Casks need to worry about. So to let you know, you may need to do some extra work beyond updating the hashes and version number once 13.5 stabilizes in a few months. If you don't mind, could you send me your email ( richard@torproject.org ) so I can keep you abreast of our potentially breaking changes in the future?

Thanks!

### Relevant casks

Mullvad Browser, Tor Browser

**Comment by miccal** [comment]
I have already fixed this in the `tor-browser-alpha` Cask here: https://github.com/Homebrew/homebrew-cask-versions/pull/19501
